submission/proxy_test.go

```diff
@@ -49,7 +49,7 @@ func stubLogListManager() *LogListManager {
 }
 
 func TestProxyRefreshLLErr(t *testing.T) {
-	p := NewProxy(stubLogListManager(), GetDistributorBuilder(ChromeCTPolicy, NewStubLogClient, monitoring.InertMetricFactory{}))
+	p := NewProxy(stubLogListManager(), GetDistributorBuilder(ChromeCTPolicy, NewStubLogClient, monitoring.InertMetricFactory{}), monitoring.InertMetricFactory{})
```

Comment:
Can these two inert metric factory instances be identical?
i.e., outside this test func, declare:

`var imf monitoring.InertMetricFactory`

and just pass `imf` in each of these tests wherever you currently initialise and pass an empty `InertMetricFactory`.

Reply:
Done